odtp.yml

```diff
@@ -53,4 +64,5 @@ schema-output: null
 
 # If applicable, define devices needed such as GPU.
 devices:
-  gpu: false
+  - type: gpu
+    required: false
```

Comment:
Hello @sabinem, the last empty line is missing. Besides that. It's fine.

Reply:
@caviri No I don't think so. There is an empty line at the bottom of the file. I merge it.